inference-engine/tests/functional/plugin/gpu/shared_tests_instances/low_precision_transformations/unsqueeze_transformation.cpp

```diff
@@ -24,27 +24,27 @@ namespace {
 
     const std::vector<LayerTestsDefinitions::UnsqueezeTransformationParam> params = {
         {
-            { 256ul, ngraph::Shape { 1, 1, 1, 1 }, { 0.f }, { 255.f }, { -128.f }, { 127.f } },
+            { 256ul, ngraph::Shape { 1, 1, 1 }, { 0.f }, { 255.f }, { -128.f }, { 127.f } },
             { 0.0, 3.0 },
             { 3, 3, 5}
         },
         {
-            { 256ul, ngraph::Shape { 1, 1, 1, 1 }, { 0.f }, { 255.f }, { -128.f }, { 127.f } },
+            { 256ul, ngraph::Shape { 1, 1, 1 }, { 0.f }, { 255.f }, { -128.f }, { 127.f } },
```

Comment:
Why did you changed test parameters?

Reply:
These parameters are invalid for Fake Quantize since according to spec tensors with ranges have to broadcastable to tensor with data. In this case rank of tensors with ranges greater than the rank of data tensor which formally means that data tensor will be broadcasted to tensor with ranges. This reflected in the reference implementation for FQ - there is a check that tensors on ports 1-4 have rank less or equal to tensor on port 0